pkg/features/kube_features.go

```diff
@@ -98,7 +94,6 @@ func init() {
 // To add a new feature, define a key for it above and add it here. The features will be
 // available throughout Kubernetes binaries.
 var defaultKubernetesFeatureGates = map[utilfeature.Feature]utilfeature.FeatureSpec{
-	ExternalTrafficLocalOnly:                    {Default: true, PreRelease: utilfeature.GA},
```

Comment:
This flag is complied into binary. In that case, it will hard fail, because ExternalTrafficLocalOnly is not a valid entry anymore.